packages/stark-ui/src/modules/table/components/table.component.html

```diff
@@ -1,68 +1,64 @@
 <div class="header">
-	<h3>Table title</h3>
-	<div class="actions">
-		<button *ngIf="isMultiSortEnabled" (click)="openMultiSortDialog()" mat-icon-button>
-			<mat-icon svgIcon="sort" matTooltip="Multi-Column Sorting"></mat-icon>
+	<div class="transcluded">
+		<ng-content select="header"></ng-content>
+	</div>
+	<div class="actions" [ngClass]="{'actions-alt': customTableActionsType === 'alt'}">
+		<stark-action-bar [actionBarConfig]="customTableActionsType === 'regular' ?{actions: customTableActions}:{}" [alternativeActions]="customTableActionsType === 'alt' ?customTableActions:null" buttonColor="alt" mode="compact"></stark-action-bar>
```

Comment:
Same remark as below:
Well yeah we could have a more flexible approach but this one is the one that cost the less effort regarding the old API.

Reply:
I understand the reason of this code, but the point here is that this code should be in the component class instead of the template. It is safer there and more difficult to overlook potential issues related to the interface that should be implemented.